### DISCUSSION TITLE
Why are pixel_{w,h} functions not part of WindowExt?

### DISCUSSION THREAD

**Discussion by abau** [fltk-rs/fltk-rs#1412] [Q&A] [answered]
Since every window seems to provide `pixel_w` and `pixel_h` functions, would it make sense to add them to `WindowExt`?

**Comment by MoAlyousef** [comment, marked as answer]
OverlayWindow and MenuWindow don’t provide them unfortunately. 
Although we could probably use the same implementation as DoubleWindow. 
Historically it was because it was only provided for OpenGL windows.

**Reply by abau** [reply]
Ah, I've missed these. Thanks!
